From 796cb29261dd13128538311c1d2680dcb938e716 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alejandro=20S=C3=A1ez?= <asm@redhat.com>
Date: Mon, 6 Nov 2023 18:59:02 +0100
Subject: [PATCH] Make it fail

---
 src/crypto/aes/aes_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/crypto/aes/aes_test.go b/src/crypto/aes/aes_test.go
index 1e8bac4bb5..0c03e23b48 100644
--- a/src/crypto/aes/aes_test.go
+++ b/src/crypto/aes/aes_test.go
@@ -13,6 +13,7 @@ import (
 // Test that powx is initialized correctly.
 // (Can adapt this code to generate it too.)
 func TestPowx(t *testing.T) {
+	t.FailNow()
 	p := 1
 	for i := 0; i < len(powx); i++ {
 		if powx[i] != byte(p) {
-- 
2.41.0

